### PR TITLE
remove opinionated operators

### PR DESCRIPTION
**Pull request by staltz**
To be merged simultaneously with https://github.com/ssb-ngi-pointer/ssb-db2/pull/10

### CODE REVIEW

**Comment by staltz** [comment]
@mixmix Check the README patches in this PR. should address #22

File: README.md [hunk 1]
```diff
@@ -104,11 +104,26 @@ query(
...
 )
+
+// The `seekType` function takes a buffer and uses `bipf` APIs to search for
+// the fields we want. Note that `~p` returns true for both `0` and `1` but
+// false for `-1`:
+const bValue = Buffer.from('value') // better for performance if defined outside
+const bContent = Buffer.from('content')
+const bType = Buffer.from('type')
+function seekType(buffer) {
+  var p = 0
```
Comment (mixmix) [inline README.md:120]: Maybe leave a comment saying
// position in the buffer, offset from start

File: README.md [hunk 2]
```diff
@@ -104,11 +104,26 @@ query(
...
-  and(equal(db.seekType, 'post', 'type')),
+  and(equal(seekType, 'post', 'type')),
   toCallback((err, msgs) => {
     console.log('There are ' + msgs.length + ' messages of type "post"')
   })
 )
+
+// The `seekType` function takes a buffer and uses `bipf` APIs to search for
+// the fields we want. Note that `~p` returns true for both `0` and `1` but
+// false for `-1`:
```
Comment (mixmix) [inline README.md:115]: Why not just use (p > -1)
Everyone knows what that means, and it's the and saves one character over (!~p)

**Review by mixmix — COMMENTED** [review]
This feels really accessible, thanks for this.
If it makes the readme too heavy I think you could just use the helpers for most examples and show the source of the seekType.

File: README.md (hunk 2 above)
Comment (mixmix) [reply README.md:115]: Wow middle of the night comments have shonky grammer

Comment (arj03) [reply README.md:115]: There is no perf difference between `if(~p) return` and `if (p == -1) return`. So yeah probably go for the more readable version.

**Review by arj03 — APPROVED** [review]
Looks good. I likes mix's comments and had a small comment on the other PR that goes together with this. Overall minor thing so good to merge from my point of view. Great work simplifying this.

**Comment by staltz** [comment]
@arj03 branch async-operators is pushed already, can you make a PR for it (for yourself to review)? :)

**Comment by arj03** [comment]
Uhhh exciting @staltz!

**Pull request merged**